.github/workflows/positron-merge-to-branch.yml

```diff
@@ -182,15 +182,23 @@ jobs:
           run: DISPLAY=:10 ./scripts/test-web-integration.sh --browser chromium
 
   slack-notify:
-    if: ${{ failure() && inputs.e2e_grep == '' }}
+    # if: ${{ failure() && inputs.e2e_grep == '' }}
+    if: ${{ inputs.e2e_grep == '' }}
     needs: [unit-tests, integration-tests, e2e-electron]
     runs-on: ubuntu-latest
     steps:
       - name: Send Slack Notification
         uses: midleman/slack-workflow-status@master
         with:
           repo_token: ${{ secrets.POSITRON_GITHUB_PAT }}
-          slack_webhook_url: ${{ secrets.SLACK_TEST_RESULTS_WEBHOOK_URL}}
+          slack_webhook_url: ${{ secrets.SLACK_TEST_RESULTS_WEBHOOK_URL }}
           notify_on: "failure"
+      # adding this for time being so i can monitor slack notifications to make sure
+      # behavior is as expected.
+      - name: Send Slack Notification
+        uses: midleman/slack-workflow-status@master
+        with:
+          repo_token: ${{ secrets.POSITRON_GITHUB_PAT }}
+          slack_webhook_url: ${{ secrets.SLACK_MARIE_WEBHOOK_URL }}
```

Comment:
i just want to keep an eye on slack notifications since i recently made some changes. will remove when done of course.